widget/playground/src/constants/presets.ts

```diff
@@ -42,20 +42,41 @@ export const PRESETS: {
       neutral: '#E6E6E6',
       background: '#FDFDFD',
       foreground: '#010101',
+      info: '#5BABFF',
     },
   },
   {
     id: 2,
+    dark: {
+      neutral: '#161C38',
+      primary: '#1C3CF1',
+      secondary: '#2284ED',
+      background: '#070917',
+      foreground: '#FDFDFD',
+      info: '#5BABFF',
+    },
     light: {
       primary: '#1C3CF1',
       secondary: '#469BF5',
       neutral: '#E6E6E6',
       background: '#FDFDFD',
       foreground: '#010101',
+      info: '#5BABFF',
     },
   },
```

Comment:
Not very important but it was a bit strange for me that two last light presets had somehow a dark background.

![Recording 2023-11-15 at 11 41 06](https://github.com/rango-exchange/rango-client/assets/99180934/100d5638-9865-436e-825b-74fc03920a23)

Reply:
fixed